File: openapi-test-kotlin-server-quarkus-reactive-client-mp-reactive/testapi/model/Owner.kt
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import jakarta.validation.constraints.NotNull
import java.util.Optional

data class Owner(
  @JsonProperty("address") val address: Optional<Address>,
  @JsonProperty("email") @jakarta.validation.constraints.Email val email: Optional<String>,
  @JsonProperty("id") @NotNull val id: String,
  @JsonProperty("name") @NotNull val name: String
)

File: openapi-test-scala-server-spring-client-jdk/testapi/model/Owner.scala
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Optional

case class Owner(
    @JsonProperty("address") address: Optional[Address],
    @JsonProperty("email") email: Optional[String],
    @JsonProperty("id") id: String,
    @JsonProperty("name") name: String
)
